packages/teleterm/src/ui/commandLauncher.test.ts

```diff
@@ -0,0 +1,36 @@
+import { MockAppContext } from 'teleterm/ui/fixtures/mocks';
+
+import { CommandLauncher } from './commandLauncher';
+
+it('returns tsh install & uninstall autocomplete command on macOS', () => {
+  const appContext = new MockAppContext({ platform: 'darwin' });
+  const commandLauncher = new CommandLauncher(appContext);
+  const autocompleteCommandNames = commandLauncher
+    .getAutocompleteCommands()
+    .map(c => c.name);
+
+  expect(autocompleteCommandNames).toContain('autocomplete.tsh-install');
+  expect(autocompleteCommandNames).toContain('autocomplete.tsh-uninstall');
+});
+
+it('does not return tsh install & uninstall autocomplete command on Linux', () => {
+  const appContext = new MockAppContext({ platform: 'linux' });
+  const commandLauncher = new CommandLauncher(appContext);
+  const autocompleteCommandNames = commandLauncher
+    .getAutocompleteCommands()
+    .map(c => c.name);
+
+  expect(autocompleteCommandNames).not.toContain('autocomplete.tsh-install');
+  expect(autocompleteCommandNames).not.toContain('autocomplete.tsh-uninstall');
+});
+
+it('does not return tsh install & uninstall autocomplete command on Windows', () => {
+  const appContext = new MockAppContext({ platform: 'win32' });
+  const commandLauncher = new CommandLauncher(appContext);
+  const autocompleteCommandNames = commandLauncher
+    .getAutocompleteCommands()
+    .map(c => c.name);
+
+  expect(autocompleteCommandNames).not.toContain('autocomplete.tsh-install');
+  expect(autocompleteCommandNames).not.toContain('autocomplete.tsh-uninstall');
+});
```

Comment:
> We can avoid repeating the same test code with [`test.each`](https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m).

I think I'm actually going to keep it, it's just three tests and idk how to neatly refactor this with `test.each` without introducing additional logic since one test requires commands to be in the array and others do not.

Reply:
Then I'd say it's not worth it for just two short tests. 😏